src/main/java/hec/ensemble/EnsembleUtility.java

```diff
@@ -94,17 +94,20 @@ static void readModifyWrite(String fileName) throws Exception {
         int count = 0;
```

Comment:
Can you tell me the difference in the timing between the previous set of code before iterator, ensembletimeseriesreader implementation of iterator, and the ensembletimeseries iterator for this test?

Reply:
commit|read(s)|write(s)|
--|--|-|
before |1.5| 5.7 |
lazy reader| 6| 5.8 |  
now | 1.5| 5.8 |

Appveyor numbers can vary , perhaps based on server load.  The write time changed from 7.3 to 5.8 with just a rebuild. Lazy reader numbers are from my memory.

Reply:
this is good enough for this situation. obviously the efficency of reading the entire dataset with one transaction is worth it.